src/activities/ActivityUsageEntity.js

```diff
@@ -60,6 +60,10 @@ export class ActivityUsageEntity extends Entity {
 	 */
 	canEditReleaseConditions() {
 
+		if (!this._entity) {
```

Comment:
You can condense this like https://github.com/BrightspaceHypermediaComponents/siren-sdk/blob/f12c4f907578baea38e24e61d9b93e22b64820af/src/organizations/OrganizationEntity.js#L28

ideally we'd be using https://github.com/tc39/proposal-optional-chaining to make that check not a thing, but you know how it is...